test/mtl/test_trainer.py

```diff
@@ -30,21 +29,22 @@ def test_trainer_onetask(self):
         dataloaders = create_dataloaders(num_tasks=1)
         scores = model.score(dataloaders)
         self.assertLess(scores["task1/TestData/test/accuracy"], 0.7)
-        trainer = Trainer(**self.trainer_config, **self.logger_config)
+        trainer = Trainer(**self.trainer_config)
         trainer.train_model(model, dataloaders)
         scores = model.score(dataloaders)
         self.assertGreater(scores["task1/TestData/test/accuracy"], 0.9)
 
     def test_trainer_twotask(self):
         """Train a model with overlapping modules and flows"""
+        set_seed(SEED)
```

Comment:
Does this not pass if the seed isn't the same? Probably not a valid unit test in that case.

Reply:
Yeah, there's variation. Will address when we make add real unit tests for slicing.